test/api.test.js

```diff
@@ -422,6 +422,72 @@ test('Draw.modes', t => {
   t.end();
 });
 
+test('Draw.combineFeatures -- multiploygon', t => {
+  const [polygonId] = Draw.add(getGeoJSON('polygon'));
+  const [polygon2Id] = Draw.add(getGeoJSON('polygon2'));
+  Draw.changeMode('simple_select', { featureIds: [polygonId, polygon2Id]});
+
+  Draw.combineFeatures();
+  t.equals(Draw.getAll().features.length, 1, 'can merge two features');
+  t.equals(Draw.getAll().features[0].geometry.type, 'MultiPolygon', 'can merge two polygons into MultiPolygon');
+  Draw.deleteAll();
```

Comment:
Oh, sorry, just seeing this. Lets tests that the coordinates from the original features match those in the multifeature